library/src/components/CollapseButton.tsx

```diff
@@ -1,11 +1,30 @@
 import React, { ButtonHTMLAttributes, SVGAttributes } from 'react';
-import { HiChevronRight } from 'react-icons/hi';
 
 interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
   chevronProps?: SVGAttributes<SVGElement>;
   expanded?: boolean;
 }
 
+const HiChevronRight = (props: SVGAttributes<SVGElement> = {}) => (
+  // Copied from https://icon-sets.iconify.design/heroicons/chevron-right/
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="1em"
+    height="1em"
+    viewBox="0 0 24 24"
+    {...props}
+  >
+    <path
+      fill="none"
+      stroke="currentColor"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+      strokeWidth="1.5"
+      d="m8.25 4.5l7.5 7.5l-7.5 7.5"
+    />
+  </svg>
```

Comment:
Change it to the:

```suggestion
  <svg 
    stroke="currentColor" 
    fill="currentColor" 
    strokeWidth="0" 
    viewBox="0 0 20 20" 
    height="1em" 
    width="1em" 
    xmlns="http://www.w3.org/2000/svg"
    {...props}
  >
    <path 
      fillRule="evenodd" 
      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z" 
      clipRule="evenodd"
    />
  </svg>
```

Reply:
Yes, react-icons [used the v1](https://github.com/react-icons/react-icons/blob/master/packages/react-icons/src/icons/index.ts#L417-L446) of HeroIcons and I added v2.